core_v14/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "core_v14"
```

Comment:
Might not be the best name if this crate could one day be used on its own? `desub-metadata-v14`?